### PR TITLE
Open an Inquiry Form on About page

### PR DESCRIPTION
**Pull request by hkurokawa**
## Related Issue
#27 Add Inquiry Url
[DroidKaigi/2016 - #151 一般参加者向けアンケートを作る](https://github.com/DroidKaigi/2016/issues/151)

## Contents
- [x] Add a URL of the form in strings.xml (Add it as a string resource in case it might change depending on the language)
- [x] Open the URL when user taps Questionnaire link on About

## Verification
- [x] Android 4.2 (Emulator) → Open the page with an external browser
- [x] Android 6.0 (Nexus 5X) → Open the page with a custom Chrome tab

[Review on Reviewable](https://reviewable.io/reviews/konifar/droidkaigi2016/71)

### CODE REVIEW

**Comment by hkurokawa** [comment]
@konifar When you have time, could you review this PR?

**Comment by hotchemi** [comment]
LGTM

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -58,6 +58,7 @@
     <string name="about_license">License</string>
     <string name="about_questionnaire">Questionnaire</string>
     <string name="about_version_prefix">"V%1$s</string>
+    <string name="about_inquiry_url">https://docs.google.com/forms/d/1fpxZOqprZyzBgMBb7HCCrq538oOzfzWkOMpItRqF-ro/viewform</string>
```
Comment (konifar) [inline app/src/main/res/values/strings.xml]: I think this should be written in Java code  because this is not translation text.
How do you think?

Comment (hkurokawa) [inline app/src/main/res/values/strings.xml]: I am afraid the form in English might have a different URL.  In such case, we can switch the URL based on the user's language if it is in string resource.

**Comment by konifar** [comment]
LGTM!!!

**Pull request merged**